src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,56 @@
+.clock {
+  @include circle(200px);
+  position: absolute;
```

Comment:
Do not use positioning props for BEM block (you can use only `position: relative` if some of child elements have `position: absolute` in case if it's used only for that purpose)